src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {
+    YELLOW, GREEN, RED, BLUE, BLACK, WHITE
```

Comment:
maybe not a mistake, but in my opinion, it is better to put each on the new line